keycloak/keycloak-admin-ui
# Create attribute - delete validator

## Motivation
Added deleting validator

## Verification Steps
1. Go to  realm-settings, user profile tab
2. Click create attribute btn, add any 2 validators and remove 1 of them and verify that removed validator is not listed in the validations table
3. Click save

## Checklist:
